experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -254,8 +220,48 @@ function invertMark<TNodeChange>(
 
 			return applyMovedChanges(invertedMark, revision, crossFieldManager);
 		}
+		case "Transient": {
+			if (mark.cellId === undefined) {
+				return [];
+			}
+
+			assert(
+				mark.detach.type === "Delete" && mark.attach.type === "Insert",
+				"TODO: Handle transient moves",
+			);
```

Comment:
This assert will also be triggered with MoveIn & Delete as well as Insert & MoveOut/ReturnFrom, right?
I'm suprised this isn't causing more tests to fail.

Reply:
I guess I forgot to test this after I added support for transient move effects.